cogs/SquadQueue.py

```diff
@@ -609,13 +768,13 @@ async def reset_bot(self, interaction: discord.Interaction):
     @commands.guild_only()
     async def schedule_sq_times(self, ctx, timestamps: commands.Greedy[int]):
         """Saves a list of sq times to skip over.  Input a list of unix utc timestamps.  Staff use only."""
-        if not await self.has_roles(ctx.author, ctx.guild.id, ctx.bot.config):
+        if not self.is_staff(ctx.author) and not await self.bot.is_owner(ctx.author):
```

Comment:
Why check for staff at all? Server admins should disable the slash command for anyone who shouldn't use it.

Reply:
its not a slash command, there was some fuckin reason idk theres probably not a good one anymore lol.  anyways only cynda really uses this so it doesnt need to be that great as long as he understands how to use it